libs/net/models/Areas/Admin/DataLocation/ConnectionModel.cs

```diff
@@ -5,7 +5,7 @@
 namespace TNO.API.Areas.Admin.Models.DataLocation;
 
 /// <summary>
-/// ConnectionModel class, provides a model that represents an product.
+/// ConnectionModel class, provides a model that represents an connection.
```

Comment:
A extra benefit of this refactor, fixing comments :D

Reply:
my pleasure